### PR TITLE
login system spec fix

### PR DESCRIPTION
**Pull request by JordanNguyen**
Updated alok and spencer's branch so that the specs pass. accept request is also functional at /requests/accept/id. requests_spect.rb contains a test of a whole workflow that includes registering a user, logging in, creating a request with auth token, and accepting the request with auth token

### CODE REVIEW

File: dispatchr-ruby/config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
-  match '/users/login/' => 'users#login', :via => :get
+  # match '/users/login/' => 'users#login', :via => :get
   match '/requests/user/' => 'requests#search_user', :via => :get
   match '/request_items/request/' => 'request_items#search_request', :via => :get
   resources :items, :requests, :users, :request_items, :addresses
+  post '/users' => 'users#create'
+  post '/login' => 'sessions#create'
+  delete '/logout' => 'sessions#destroy'
+  post '/sign_up' => 'registrations#create'
+  match '/requests/accept/' => 'requests#accept_request', :via => :post
```
Comment (spencerprescott) [inline dispatchr-ruby/config/routes.rb]: Lets make this look like this
```ruby
post '/requests/accept/:id', to: 'requests#accept_request'
```
That way i can request with the url like this "http://localhost:3000/requests/accept/1" and not have to stick the id in the post json data.

**Comment by JordanNguyen** [comment]
[Sessions Implementation](https://trello.com/c/bva0Xewo/30-sessions-implementation)

**Comment by JordanNguyen** [comment]
[API endpoint to complete a request](https://trello.com/c/vLST2FK1/54-api-endpoint-to-complete-a-request)

**Comment by JordanNguyen** [comment]
[API endpoint to accept a request](https://trello.com/c/9jYcz4dj/51-api-endpoint-to-accept-a-request)

**Comment by JordanNguyen** [comment]
[Add a 'request status' and 'hero_id' to Request table](https://trello.com/c/htRUgSiY/31-add-a-request-status-and-hero-id-to-request-table)

**Pull request merged**